code/game/machinery/doors/airlock_types.dm

```diff
@@ -2,45 +2,44 @@
 /obj/structure/machinery/door/airlock/secure/colony
 	name = "\improper Secure Airlock"
 	icon = 'icons/obj/structures/doors/personaldoor.dmi'
-	assembly_type = /obj/structure/airlock_assembly/airlock_assembly_com
+	assembly_type = /obj/structure/airlock_assembly
 	openspeed = 4
 	req_one_access = list(ACCESS_CIVILIAN_PUBLIC)
 
 /obj/structure/machinery/door/airlock/command/colony
 	name = "\improper Command Airlock"
 	icon = 'icons/obj/structures/doors/comdoor.dmi'
-	assembly_type = /obj/structure/airlock_assembly/airlock_assembly_com
 	req_one_access = list(ACCESS_CIVILIAN_BRIG, ACCESS_CIVILIAN_COMMAND, ACCESS_WY_COLONIAL)
 
 /obj/structure/machinery/door/airlock/security/colony
 	name = "\improper Security Airlock"
 	icon = 'icons/obj/structures/doors/secdoor.dmi'
-	assembly_type = /obj/structure/airlock_assembly/airlock_assembly_sec
+	assembly_type = /obj/structure/airlock_assembly
```

Comment:
```suggestion
	assembly_type = /obj/structure/airlock_assembly
```
This entire line for all of these isn't needed, indeed assembly_type variable as a whole can probably be removed if you're removing all subtypes of it.